src/Controller/AdministrationController.php

```diff
@@ -45,7 +45,7 @@ public function addAdminFile(Request $request, EntityManagerInterface $entityMan
         $this->addFlash('failAdministration', 'Il y a eu un problème dans la mise en ligne de vos informations.');
 
         return $this->render('adminFile/index-file.html.twig', [
```

Comment:
Pour éviter les erreurs de nom de dossier mettez ça : 
```suggestion
        return $this->render('AdminFile/index-file.html.twig', [
```